_posts/meetups/2019-02-26-nlp_in_practice.md

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
The date in the file name should be 2019-12-11 (if the date from the file is picked up for the menu item)

Reply:
Yep, also fixed the meetups that said they were in 2018